src/features/courses/course-route-validation.ts

```diff
@@ -102,13 +102,22 @@ export const updateCourseTopicValidation = {
 export const extendCourseTopicForUserValidation = {
     params: {},
     body: {
-        startDate: Joi.date().optional(),
-        endDate: Joi.date().optional(),
-        deadDate: Joi.date().optional()
+        extensions: Joi.object({
+            startDate: Joi.date().optional(),
```

Comment:
The problems you caught with us putting moments in the database didn't originate from Joi, right? If we hit these endpoints with moment from the frontend, everything is still fine?

Reply:
* Correct, the issue @doombot-exe and I saw last night was the backend sending joi along
* I did not see an error on the frontend
* Furthermore a frontend type can't push through to the backend due to serialization and deserialization, if this was a problem joi date would have failed